packages/components/src/Banner/Banner.tsx

```diff
@@ -25,20 +41,34 @@ export function Banner({ children, type, onDismiss }: BannerProps) {
     error: "red",
   };
 
+  if (primaryAction != undefined) {
+    primaryAction = Object.assign(
+      {
+        size: "small",
+        type: "tertiary",
+        variation: type === "notice" ? "learning" : "work",
```

Comment:
What is this setting exactly?

Reply:
Added a description to this variable. Do you think that will suffice?

Reply:
Might be better to add this as a documentation on the `type` prop so that you can see it in the docs viewer as well?

Reply:
I have added it to the documentation of `primaryAction` I don't think that type is the right place for it as it doesn't change anything about type.